globality_black/blank_lines.py

```diff
@@ -28,10 +27,9 @@ def cover_blank_lines(module, root):
 
 
 def add_token_if_line_to_keep(prefix):
-    # Add a token when two new lines
-
+    # Add a token when two new lines and a comment
     return re.sub(
-        r"\n{2}( +)",
+        r"\n{2,}(\s+\# .+\n\s+)",
```

Comment:
Why are we changing behavior and no longer protecting single empty lines?

I thought the problem was that we were leaving multiple empty lines, wouldn't this be a solution?
```suggestion
        r"\n{2,}( +)",
```